rts/motoko-rts/src/constants.rs

```diff
@@ -1,21 +1,10 @@
-use crate::types::{Bytes, Words};
+use crate::types::Bytes;
 
 /// Wasm word size. RTS only works correctly on platforms with this word size.
-pub const WORD_SIZE: u32 = 4;
-
-/// Maximum Motoko array size 2^29 (inclusive)
-/// NB: Must agree with Arr.max_array_size in compile.ml.
-pub const MAX_ARRAY_SIZE: u32 = 1 << 29;
```

Comment:
Is MAX_ARRAY_SIZE bigger now?

Reply:
There is a new `MAX_ARRAY_LENGTH_FOR_ITERATOR` constant in `memory.rs` that I will move to this place.

The maximum length is substantially larger now, 2**61, only limited by the iterator logic that needs to fit the array length inside a compact number. Technically, as long as we do not have specialized array layouts, the address space is the actual limit, since the maximum length times the standard 64-bit element size is even larger than 64-bit.